### PR TITLE
PR: Range Slider

### PR DESCRIPTION
**Pull request by Jayesh3636**
Added Range SlideR

https://user-images.githubusercontent.com/91378157/157523953-5aa97b44-bd9c-4602-9ae4-06e9ffef4d4d.mp4

Hey guys kindly Review the slider!
Let me know the changes if required.
TIA~!

### CODE REVIEW

File: Components/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+.label-active{
+    color: var(--primary-color);
+}
+
+.range-slider [type="range"] {
+    width: 100%;
+    -webkit-appearance: none;
+    height: 2px;
+    border-radius: 2px;
+    background: #ffffff;
```
Comment (sharmayatendra) [inline Components/slider/slider.css]: use color variables

File: Components/slider/slider.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    background: var(--primary-color);
+}
+
+/* for mozillaaaa */
+.range-slider [type="range"]::-moz-range-thumb {
+    width: 15px;
+    height: 15px;
+    border: 0;
+    border-radius: 50%;
+    background: #ffffff;
```
Comment (sharmayatendra) [inline Components/slider/slider.css]: same as above

**Review by sharmayatendra — APPROVED** [review]
can make changes as written. good work though

File: Components/slider/slider.css (hunk 1 above)
Comment (Jayesh3636) [reply Components/slider/slider.css]: Done! Thanks!

File: Components/slider/slider.css (hunk 2 above)
Comment (Jayesh3636) [reply Components/slider/slider.css]: Done! Thanks!

**Comment by AshwiniSetty** [comment]
Good Work

**Pull request merged**